docs/dev_building_web_assembly.rst

```diff
@@ -0,0 +1,146 @@
+
+=========================================
+Building libCellML for use in the Browser
+=========================================
+
+To build libCellML for the browser we use the emscripten toolchain to create web assembly binaries.
+Here we will go through the process of building web assembly binaries on macOS.
+For other operating systems modifications to these instructions will have to be made.
```

Comment:
To build libCellML for the browser, we use the emscripten toolchain to create web assembly binaries.
Here, we will go through the process of building web assembly binaries on macOS.
For other operating systems, modifications to these instructions will have to be made.

Reply:
9edec02